<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DDGClass</class>
 <widget class="QMainWindow" name="DDGClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>494</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>水滴数据生成器Ver 1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../DDG.qrc">
    <normaloff>:/DDG/Resources/datag.ico</normaloff>:/DDG/Resources/datag.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>21</y>
       <width>268</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>文件名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editflname">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>test</string>
        </property>
        <property name="placeholderText">
         <string>test</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>189</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>文件个数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editflnum">
        <property name="text">
         <string>10</string>
        </property>
        <property name="placeholderText">
         <string>10</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>生成数据类型</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>20</y>
       <width>262</width>
       <height>18</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>整数</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>实数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>英文字符</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_7">
        <property name="text">
         <string>英文+数字</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>分隔方式</string>
    </property>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>无</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>回车</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>空格</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnMake">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>数据属性</string>
    </property>
    <widget class="QLineEdit" name="editnumrange">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="placeholderText">
      <string>0,100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>个数范围</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editdatarange">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>0,100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>大小范围</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>*如果是单个数，就是个固定值；两个数用逗号分开，表示范围</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>生成情况：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbout">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>说明</string>
    </property>
   </widget>
   <zorder>btnAbout</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>btnMake</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>lblStatus</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../DDG.qrc"/>
 </resources>
 <connections/>
</ui>
